modules/zero-trust-manager-enable-metrics-server.adoc

```diff
@@ -6,68 +6,197 @@
 [id="zero-trust-manager-enable-metrics-server_{context}"]
 = Configuring metrics collection for SPIRE Server by using a Service Monitor
 
+[role="_abstract"]
+The {zero-trust-full} exposes metrics by default on port 8443 at the `/metrics` service endpoint. You can configure metrics collection for the Operator by creating a `ServiceMonitor` custom resource (CR) that enables the Prometheus Operator to collect custom metrics. For more information, see "Configuring user workload monitoring".
+
+
 The SPIRE Server operand exposes metrics by default on port `9402` at the `/metrics` endpoint. You can configure metrics collection for the SPIRE Server by creating a `ServiceMonitor` custom resource (CR) that enables the Prometheus Operator to collect custom metrics.
 
+
 .Prerequisites
 
 * You have access to the cluster as a user with the `cluster-admin` cluster role.
 
 * You have installed the {zero-trust-full}.
 
-* You have deployed the SPIRE Server operand in the cluster.
-
 * You have enabled the user workload monitoring.
 
 .Procedure
 
-. Create the `ServiceMonitor` CR:
+. Configure the Operator to use HTTP or HTTPS protocols for the metrics server.
+
+.. Update the subscription object for {zero-trust-full} to configure the HTTP protocol by running the following command:
++
+[source,terminal]
+----
+$ oc -n zero-trust-workload-identity-manager patch subscription zero-trust-workload-identity-manager-subscription --type='merge' -p '{"spec":{"config":{"env":[{"name":"METRICS_BIND_ADDRESS","value":":8080"}, {"name": "METRICS_SECURE", "value": "false"}]}}}'
+----
+
+.. Verify the {zero-trust-full} pod is redeployed and that the configured values for `METRICS_BIND_ADDRESS` and `METRICS_SECURE` is updated by running the following command:
++
+[source,terminal]
+----
+$ oc set env --list deployment/zero-trust-workload-identity-manager-controller-manager -n zero-trust-workload-identity-manager | grep -e METRICS_BIND_ADDRESS -e METRICS_SECURE -e container
+----
++
+.Example output
+[source,text]
+----
+deployments/zero-trust-workload-identity-manager-controller-manager, container manager
+METRICS_BIND_ADDRESS=:8080
+METRICS_SECURE=false
+----
+
+. Create the Secret resource with `kubernetes.io/service-account.name` annotation to inject the token required for authenticating with the metrics server.
```

Comment:
Out of scope for merge review, but if you're referencing an actual object, it should be marked up appropriately in backticks, like `Secret`. Here and a few other places